jellyfin/jellyfin-androidtv
# Refactor CustomPlaybackTransportControlGlue to extract logic into actions

**Changes**
Refactors the `CustomPlaybackTransportControlGlue` by:
- Removing static dispatch for custom actions and built-in actions
- Encapsulating the logic for each action into the action class

This will make it much easier to unit test actions, because we only need to inject the mocks for the method signature rather than everything the glue class could possibly need.

I've done this with as minimal changes to `videoAdapter` to not interfere with any playback rewrites going on. It's not perfect but (hopefully) it creates so better separation of concerns that will allow us to write tests at a later date.

In the future I want to shift as much logic out of the glue class into the relevant actions, but this diff is already quite large...

**To Test**
- Check all buttons work whilst playing media.
- Double check live TV buttons (as I don't have access to those options locally)
